pkg/controller/nodeconfig/controller.go

```diff
@@ -73,6 +74,15 @@ type Controller struct {
 	operatorImage string
 }
 
+func isManagedByNodeConfigController(obj kubeinterfaces.ObjectInterface) bool {
+	labels := obj.GetLabels()
+	if labels == nil {
+		return false
+	}
```

Comment:
nit: unnecessary

Reply:
many object definitions may not have the labels map initialized, this feels safe to avoid panics

Reply:
> many object definitions may not have the labels map initialized

yes, but you can safely read in that case anyway.
[`A nil map is equivalent to an empty map except that no elements may be added.`](https://go.dev/ref/spec#Map_types)

You can still prefer to do this imo, hence a nit

Reply:
not a nit, you are correct :+1:

Reply:
fixed